Software-Engineering/L4-Lead-Software-Engineer.md

```diff
@@ -5,7 +5,7 @@
 - **Planning horizon**: 6-12 months
 - **Impact radius**: Team (5-15)
 - **Evaluation**: Manager or Director
-- **Responsibility and direction needed**: Takes brand new ideas from inception to production with limited supervision.
+- **Responsibility and direction needed**: Consistently takes brand new ideas from inception to production with limited supervision.
```

Comment:
I'm not sure I fully understand the difference here between the L3
`I confidently and autonomously lead the shipping of ideas from conception to production`
and the L4 
`Consistently takes brand new ideas from inception to production with limited supervision.`

I wonder if the L3 could have some sort of qualifier, that the "idea" is a piece of a larger idea, or that it's well defined, where as for L4 it's more ambiguous?